src/domains/auth/services.ts

```diff
@@ -5,15 +5,12 @@ import { auth } from "~/services/firebase";
 
 const userCollection = collection(firestore, "users");
 
-export async function getUser(userId: string) {
-  const userDoc = doc(userCollection, userId);
-
-  const user = await userDoc.get();
-
-  return {
-    ...user.data(),
-    id: user.id,
-  };
+export async function getUser(id: string): User {
+	const doc = await getDoc(doc(usersCollection, id))
+	return {
+		id: doc.id(),
+		...doc.data()
```

Comment:
To avoid `id` overwriting, you should swap it. Please, change it in all places.

```suggestion
...doc.data(),
id: doc.id()
```